source/mongorestore.txt

```diff
@@ -706,6 +706,13 @@ Options
    To use :option:`--preserveUUID <mongorestore --preserveUUID>`, you
    must also include the :option:`--drop <mongorestore --drop>` option.
 
+   .. note::
+
+      If ``mongorestore`` uses the 
+      :option:`--preserveUUID <mongorestore--preserveUUID>` option and
```

Comment:
[nit] [style guide - avoid repeated links ](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#minimize-repeated-links)

Since this note is talking about the section it's contained in, no need to link it. We can just leave it monospaced. Same goes for the text on line 706 (though I know that isn't your work from this ticket).

Reply:
Done!